scroll-workspaces/extension.js

```diff
@@ -166,8 +166,14 @@ Ext.prototype = {
 				});
 
 			// Do not show wokspaceSwithcer in overview
-			if(!Main.overview.visible)
-				Main.wm._workspaceSwitcherPopup.display(motion, ws.index());
+			if(!Main.overview.visible) {
+				// Fix for Gnome 43+
+				if(imports.misc.config.PACKAGE_VERSION >= '42') {
```

Comment:
String comparison? That doesn't seem right, e.g. this would presumably break one day when gnome hits v100 :sweat: 

Is there an integer we can compare against easily? If not, I'm happy to restrict future releases to v42+ in metadata.json

Reply:
You're right ! I'm not used to these Gnome APIs, didn't think that though. No idea why they don't expose the major version directly as an int...

Would converting this into an int first be alright ? This should do the job, assuming they don't change the formatting down the line...

```suggestion
				if(parseInt(imports.misc.config.PACKAGE_VERSION) >= 42) {
```